Packs/ThycoticDSV/README.md

```diff
@@ -0,0 +1,5 @@
+This package contains the integration of XSOAR and Thycotic DevOps Secrets Vault.
+
+Manage credentials for applications, databases, CI/CD tools, and services without causing friction in the development process.
```

Comment:
@AndyNikolaev  please make this pack readme more complete (more like the Secret Server one) with more details, use cases, etc. Add links back to Thycotic site, too, if you want.

Reply:
Add